cni-plugin/integration/run.sh

```diff
@@ -99,6 +99,11 @@ echo 'PASS: Network Validator'
 
 calico_overrides="{
                \"apiVersion\": \"v1\",
+               \"metadata\": {
+                \"annotations\": {
+                    \"LINKERD_CNI_DEBUG_LOGFILE\": \"/var/log/linkerd-cni.log\",
```

Comment:
we'd probably want to change the format here a bit. k8s annotation follow a more strict domain convention (think JVM packages)

```
config.linkerd.io/cni-debug-logfile: "/var/log/linkerd-cni.log"
```

Could also introduce a new subdomain? `debug.linkerd.io/cni-logfile`.